pagopa/io-app
# feat(Sicilia Vola): [IASV-24] Check income worker screen

## This PR depends on #3278 
## Short description
This PR integrate the `CheckIncomeComponent` in the `WorkerCheckIncomeScreen`.

## List of changes proposed in this pull request
- Insert `CheckIncomeComponent`
- Remove unused props and imports

https://user-images.githubusercontent.com/11773070/128159491-f899d3b5-b1b6-4c40-bfbf-b01052a6c121.mp4


## How to test
Modify in the `CheckStatusRouterScreen` the following lines:
https://github.com/pagopa/io-app/blob/0150ad3c78a7b842f02d929ab88ca5b43ce0bf23/ts/features/bonus/siciliaVola/screens/voucherGeneration/CheckStatusRouterScreen.tsx#L99-L100
in:
```
isServiceAlive: remoteReady(true),
tosAccepted: remoteReady(true)
```
Starts the flow from the SiciliaVola service detail and select in:
- `CheckStatusRouterScreen` -> `Resident in Sicily`
- `SelectBeneficiaryCategoryScreen` -> `Worker`

Make sure you have the updated version of the master branch on dev-server in order to see the service details.
